index.html

```diff
@@ -7,26 +7,47 @@
 		<title>Awesome Books</title>
 	</head>
 	<body>
-		<h1>Awesome Books</h1>
-		<section id="books"></section>
+		<section id="header">
+			<h2>Awesome Books</h2>
+			<ul class="ul">
+				<li id="list" class="li">List of Awesome books</li>
+				<li id="addnew" class="li">Add new Book</li>
+				<li id="contact" class="li">Contact info</li>
+			</ul>
+		</section>
+		<section id="books">
+			<h1>All Awesome Books</h1>
+
+		</section>
 		<section id="createBook">
+			<hr id="hr">
+			<h2>Add a new book</h2>
 			<form action="" id="form">
```

Comment:
### Optional
- [ ] The add books functionality is not working from the form in the browser, instead i had to manually populate the books array
![Screenshot from 2023-07-07 23-11-30](https://github.com/Wils-FOURR-speed/Awesome-Books/assets/46388113/c69920a9-4dc1-4297-9614-eefff086ece3)
It is important to fix the functionality although that is out of scope for this project but important